extension/src/lib/chromeSettings.js

```diff
@@ -0,0 +1,143 @@
+                    highContrastTheme: {
+                        source: "{settings}.model.highContrastTheme",
+                        target: "highContrastTheme",
+                        singleTransform: {
+                            type: "fluid.transforms.valueMapper",
+                            defaultInputValue: "black-white",
+                            options: {
+                                "black-white": {
+                                    outputValue: "bw"
+                                },
+                                "white-black": {
+                                    outputValue: "wb"
+                                },
+                                "black-yellow": {
+                                    outputValue: "by"
+                                },
+                                "yellow-black": {
+                                    outputValue: "yb"
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        zoom: {
+            type: "gpii.chrome.zoom",
+            options: {
+                model: {
+                    magnifierEnabled: "{settings}.model.magnifierEnabled",
+                    magnification: "{settings}.model.magnification"
+                }
+            }
+        },
+        wsConnector: {
+            type: "gpii.wsConnector",
+            options: {
+                solutionId: "com.ilunion.cloud4chrome",  // This will change in the near future
+                flowManager: "ws://localhost:8081/browserChannel",
+                retryTime: 10
+            }
+        },
+        notifications: {
+            type: "gpii.chrome.notifications"
+        }
+    },
+    model: "{settings}.options.defaultSettings",  // Defaults
+    invokers: {
+        updateSettings: {
+            funcName: "gpii.chrome.settings.updateSettings",
+            args: ["{that}",  "{arguments}.0"]
+        }
+    },
+    listeners: {
+        "{wsConnector}.events.onSettingsChange": "{settings}.updateSettings",
+        "{chromeVox}.events.onError": {
+            funcName: "gpii.chrome.settings.handleExtensionHolderError",
+            args: ["{that}", "{chromeVox}", "{arguments}.0"]
+        }
+    }
+});
+
+gpii.chrome.settings.updateSettings = function (that, settings) {
+    that.applier.change("", settings || that.options.defaultSettings);
+};
+
+gpii.chrome.settings.handleExtensionHolderError = function (that, extension, error) {
+    if (error.message === "Failed to find extension with id " + extension.options.extensionId + ".") {
+        var options = {
+            type: "basic",
+            title: "GPII notifications",
+            message: extension.options.name + " couldn't be found. Do you want to add it from the chrome store?",
+            iconUrl: "https://raw.githubusercontent.com/javihernandez/android/master/platform/app/res/drawable-hdpi/gpii_logo.png",
```

Comment:
This is a pretty funny URL! Perhaps it would be better to source this from within the extension itself? I assume there is a scheme for bundling extension resources.